Taskfile.yml

```diff
@@ -4,6 +4,23 @@ includes:
   dist: ./DistTasks.yml
 
 tasks:
+  update-deb-copyright:
+    desc: Extract project and dependency licenses into debian copyright
+    cmds:
+      - |
+        if ! command -v licensed >/dev/null; then
+          echo "Installing Ruby gem: licensed"
+          gem install licensed
+        fi
+      - licensed notices
+      - mkdir -p debian/arduino-cli/usr/share/doc/arduino-cli/
+      - cat  debian/arduino-cli/usr/share/doc/arduino-cli/copyright.txt > debian/arduino-cli/usr/share/doc/arduino-cli/copyright
```

Comment:
the file is already there, you don't need to cat it. You should just rename it from `debian/arduino-cli/usr/share/doc/arduino-cli/copyright.txt` to `debian/arduino-cli/usr/share/doc/arduino-cli/copyright.txt` and commit the no txt version.